src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        return Color.values()[new Random().nextInt(Color.values().length)].toString();

```

Comment:
```suggestion
        return Color.values()[random.nextInt(Color.values().length)].name();
```

lets make Random as class level variable, not to create new instance of Random class every time we call getRandomColor() method